definitions/artifacts/azure-fhir-service.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$md": {
+    "access": "public",
+    "name": "azure-fhir-service"
+  },
+  "type": "object",
+  "title": "Azure FHIR Service",
+  "description": "Azure FHIR Service authentication APIs.",
+  "additionalProperties": false,
+  "required": [
+    "data"
```

Comment:
Let's add specs to required

Reply:
I was hesitant to add `specs` to this iteration because currently the `specs` for Azure are just 4 regions. However, this bundle supports more than those 4 regions. Deploying the service to one of those regions that is not in the Azure `specs` causes the artifact to fail on deployment.

Reply:
This is a great point and thanks for kicking off a discussion in bundle-discuss. In the name of iteration, feel free to merge this PR without making the change. We can act on the resolution of that thread. Thanks Michael! 👏